apps/web/src/presenters/commitPresenter.ts

```diff
@@ -0,0 +1,5 @@
+import type { Commit } from '@latitude-data/core/browser'
+
+export default function commitPresenter(commit: Commit) {
+  return commit
+}
```

Comment:
useless presenter are you planning on using it for something in a future PR? otherwise remove please